libtenzir/builtins/aggregation-functions/all.cpp

```diff
@@ -51,7 +56,40 @@ class all_function final : public aggregation_function {
   std::optional<bool> all_ = {};
 };
 
-class plugin : public virtual aggregation_function_plugin {
+class all_instance final : public aggregation_instance {
+public:
+  explicit all_instance(ast::expression expr) : expr_{std::move(expr)} {
+  }
```

Comment:
If there is no input, `all` should return `true` (and `any` should return `false`).

Reply:
Why not `null`?